src/renderer/index.tsx

```diff
@@ -4,13 +4,9 @@ import ReactDOM from 'react-dom';
 import './services/i18n';
 import AppLoadFallback from './components/app-load-fallback';
 import { isDevelopment } from './env-and-consts';
-import { globalKeyDownHandler, globalKeyUpHandler } from './services/key-event-handler-registry';
```

Comment:
Can you also delete the files at `services/key-event-handler-registry` since they are no longer used?